README.md

```diff
@@ -7,11 +7,14 @@
 [![Build Status](https://img.shields.io/travis/contentful/contentful-persistence.swift/master.svg?style=flat)](https://travis-ci.org/contentful/contentful-persistence.swift)
 [![Coverage Status](https://img.shields.io/coveralls/contentful/contentful-persistence.swift.svg)](https://coveralls.io/github/contentful/contentful-persistence.swift)
 
-Simplified persistence for the [Contentful][1] [Swift SDK][2].
+An integration to simplify persisting data from [Contentful][1] to a local CoreData database; built on top of the official Contentful [Swift SDK][2].
 
 [Contentful][1] provides a content infrastructure for digital teams to power content in websites, apps, and devices. Unlike a CMS, Contentful was built to integrate with the modern software stack. It offers a central hub for structured content, powerful management and delivery APIs, and a customizable web app that enable developers and content creators to ship digital products faster.
+
 ## Usage
 
+Before getting started, it is highly recommended that you familiarize yourself with Apple's CoreData framework as most issues encountered during development with contentful-persistence.swift are CoreData specific and not Contentful specific. Read the [CoreData Programming Guide](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/CoreData/index.htmlhttps://developer.apple.com/library/content/documentation/Cocoa/Conceptual/CoreData/index.html) and check out other (non-Contentful) examples. 
```

Comment:
The url is doubled. Or I'm drunk. Eitherway one should be fixed ... :D

Reply:
I'd probably not mention that part `as most issues encountered during development ...` just strike it, it sounds a bit too defensive for me ... :)